interfaces/html.idl

```diff
@@ -1,30 +1,9 @@
-// HTML IDLs
-typedef (Int8Array or Uint8Array or Uint8ClampedArray or
-         Int16Array or Uint16Array or
-         Int32Array or Uint32Array or
-         Float32Array or Float64Array or
-         DataView) ArrayBufferView;
```

Comment:
Oh, great point!

Shouldn't that be baked into idlharness then? (Not sure whether or not it is.)